main.py

```diff
@@ -4,28 +4,33 @@
 from nypl_py_utils.functions.config_helper import load_env_file
 
 from redis_client import RedisClient, RedisClientError
+import json
 
 logger = create_log('lambda_function')
 
 
 def handler(event, context):
     logger.info('Connecting to redis')
-    try:
-        load_env_file(os.environ['ENVIRONMENT'], 'config/{}.yaml')
-        endpoint = os.environ['REDIS_ENDPOINT']
-        redis_client = RedisClient(endpoint)
-    except RedisClientError as e:
-        logger.error('error connecting to redis: {}'.format(e))
-    try:
-        barcodes = event['queryStringParameters']['barcodes'].split(',')
-        barcodes_with_prefix = ['m2-barcode-store-by-barcode-' + barcode for barcode in barcodes]
-        response = redis_client.get_customer_codes(barcodes_with_prefix)
-        return {
-                "statusCode": response['status'],
-                "body": json.dumps(response),
-                "headers": {
-                "Content-type": "application/json"
-                }
-        }
-    except Exception as e:
-        logger.error('error getting barcodes :{}'.format(e))
\ No newline at end of file
+    if 'docs' in event['path']:
+        with open('swagger.json', 'r') as json_content:
+            return json_content.read()
```

Comment:
This is the Lambda communicating with the API Gateway, so you need to sandwich it in an object modelling the response like
```
return {
  "statusCode": 200,
  "body": ...,
  "headers": {
    "Content-type": "application/json"
  }
}
```